packages/table/stories/Table.stories.tsx

```diff
@@ -63,10 +64,48 @@ const veryLongRenderer = () => (
     </span>
   </TextCell>
 );
+
 const empty = () => <Cell>empty</Cell>;
 const width = ({ width: totalWidth }: { width?: number }) =>
   totalWidth ? totalWidth * 0.3 : 100;
 
+const itemCellRenderer = item => (
+  <Cell>
+    <span>{item}</span>
+  </Cell>
+);
+class ChangingTable extends React.Component<
```

Comment:
I suggest using a functional component that where it's props change randomly based on `setInterval`